js/src/bootstrapMaterialDesign.js

```diff
@@ -140,8 +140,8 @@ const BootstrapMaterialDesign = (($) => {
 
             // add to arrive if present and enabled
             if (document.arrive && this.config.arrive) {
-              $document.arrive(selector, (element) => {  // eslint-disable-line no-loop-func
-                $(element)[jqueryFn](componentConfig)
+              $document.arrive(selector, function(){  // eslint-disable-line no-loop-func
```

Comment:
It works well with `function`. I think use the `function` is just ok under ES2015 . `Arrivejs` just binds the element to `this`, so the `arrow function` does not work. I can't find a better solution except overwrite `arrivejs`.